network-api/networkapi/wagtailpages/pagemodels/mixin/foundation_metadata.py

```diff
@@ -72,3 +72,11 @@ def get_meta_image(self):
 
     class Meta:
         abstract = True
+
+
+# Overriding default name and help text for the promote panels 'search_image' field.
+search_image_field = FoundationMetadataPageMixin._meta.get_field("search_image")
+search_image_field.verbose_name = "Share Image"
+search_image_field.help_text = 'Image must be high quality, include our logo mark and have the dimensions ' \
+                               '1200 x 628 px. For more design guidelines see here: ' \
+                               'https://foundation.mozilla.org/en/docs/brand/brand-identity/social-media/#og-images'
```

Comment:
Since every page inherits from the `FoundationMetaDataPageMixin`, I have added the overrides to this file so the updates can be sitewide.

Reply:
Yup. I like this approach.